packages/internal/package.json

```diff
@@ -26,6 +26,8 @@
     "@graphql-codegen/typescript-operations": "2.0.1",
     "@graphql-codegen/typescript-react-apollo": "3.0.0",
     "@graphql-codegen/typescript-resolvers": "2.0.0",
+    "babel-plugin-polyfill-corejs3": "0.2.4",
+    "core-js": "3.16.3",
```

Comment:
These were added in https://github.com/redwoodjs/redwood/pull/3301 Just re-ordered here.